etc/build/dash_compatibility.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+VERSION=$("$GOPATH/bin/pachctl" version --client-only)
```

Comment:
Nit: I think at one point we had an issue calling `$GOPATH/bin/pachctl` vs just `pachctl`—I don't remember which was the problematic one, but one wasn't getting updated for some engineers and was causing issues.

Reply:
Update, this is fine—`$GOPATH/bin/pachctl` was the preferred call path